docs/rfcs/0000-new-pipeline.md

```diff
@@ -0,0 +1,252 @@
+- Start Date: 2020-11-18
+- Target Major Version: 2.0
+- Reference Issues: (leave this empty)
+- Implementation PR: (leave this empty)
+- Author: @yorkie
+
+# Summary
+
+Introducing a new Pipeline syntax for supporting:
+
+- multiple nodes for the same plugin.
+- model deployment and distribution.
+
+# Basic example
+
+```json
+{
+  "specVersion": "2.0",
+  "plugins": [
+    {
+      "type": "dataCollect",
+      "package": "@pipcook/plugins-chinese-poem-data-collect",
+      "params": {
+        "url": "foobar"
+      }
+    },
+    {
+      "type": "dataAccess",
+      "package": "@pipcook/plugins-textline-data-access"
+    },
+    {
+      "type": "dataProcess",
+      "package": "@pipcook/plugins-textline-data-process",
+      "params": {
+        "arg1": "foobar"
+      }
+    },
+    {
+      "type": "dataProcess",
+      "package": "@pipcook/plugins-textline-data-process",
+      "params": {
+        "arg2": "foobar"
+      }
+    },
+    {
+      "type": "modelDefine",
+      "package": "@pipcook/plugins-tfjs-text-lstm-model-define"
+    },
+    {
+      "type": "modelTrain",
+      "package": "@pipcook/plugins-tfjs-text-lstm-model-train",
+      "params": {
+        "epochs": 200
+      }
+    },
+    {
+      "type": "modelEvaluate",
+      "package": "@pipcook/plugins-tfjs-text-lstm-model-evaluate"
+    },
+    {
+      "type": "deploy",
```

Comment:
Is it better to specify the type of 'modelDeploy', just to be in conformity with other plugins' types

Reply:
The name "modelDeploy" is the deprecated type, I use the new and common name is that what the plugin to deploy is not only the model, includes the runtime, data processing, and others.